apps/src/componentLibrary/dropdownMenu/dropdownMenu.module.scss

```diff
@@ -0,0 +1,238 @@
+@import "color";
+@import "@cdo/apps/componentLibrary/common/styles/mixins";
+
+// Dropdown common styles
+.dropdownContainer {
+  position: relative;
+  display: inline-block;
+  margin: 0;
+
+  .dropdownLabel {
+    display: block;
+  }
+
+  .dropdown {
+    display: inline-flex;
+    align-items: center;
+    align-self: stretch;
+    width: auto;
+    background-color: unset;
+    border-radius: 0.25rem;
+    border: 1px solid;
+    color: $light_black;
+
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+
+    &:focus-visible {
+      outline: 2px solid $light_primary_500;
+      border-radius: 0.25rem;
+      outline-offset: 2px;
+    }
+  }
+
+  // custom dropdown arrow styles
+  .dropdownArrowDiv {
+    position: relative;
+
+    &:after {
+      content: "\f078";
+      font-family: "Font Awesome 6 Pro", sans-serif;
+      position: absolute;
+      pointer-events: none;
+      text-align: center;
+    }
+  }
+}
+
+// Dropdown colors
+.dropdownContainer-black {
+  .dropdownLabel {
+    color: $light_black;
+  }
+
+  .dropdown {
+    color: $light_black;
+    border-color: $light_black;
+  }
+
+  .dropdownArrowDiv:after {
+    color: $light_black;
+  }
+
+  &:has(.dropdown:hover) {
+    .dropdown:not(:disabled) {
+      color: $light_black;
+      background-color: $light_gray_100;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_black;
+    }
+  }
+
+  &:has(.dropdown:active) {
+    .dropdown:not(:disabled) {
+      color: $light_black;
+      background-color: unset;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_black;
+    }
+  }
+
+  &:has(.dropdown:disabled) {
+    .dropdown {
+      color: $light_gray_200;
+      border-color: $light_gray_200;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_gray_200;
+    }
+  }
+}
+
+.dropdownContainer-white {
+  .dropdownLabel {
+    color: $light_white;
+  }
+
+  .dropdown {
+    color: $light_white;
+    border-color: $light_white;
+  }
+
+  &:has(.dropdown:hover) {
+    .dropdown:not(:disabled) {
+      color: $light_white;
+      background-color: $light_gray_900;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_white;
+    }
+  }
+
+  &:has(.dropdown:active) {
+    .dropdown:not(:disabled) {
+      color: $light_white;
+      background-color: unset;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_white;
+    }
+  }
+
+  &:has(.dropdown:disabled) {
+    .dropdown {
+      color: $light_gray_900;
+      border-color: $light_gray_900;
+    }
+
+    .dropdownArrowDiv:after {
+      color: $light_gray_900;
+    }
+  }
+
+  .dropdownArrowDiv:after {
+    color: $light_white;
+  }
+}
+
+// Sizes
+.dropdownContainer-l {
```

Comment:
Right now, when we only have 1 dropdown I think it makes more sense to declare dropdown styles like this since different dropdownComponents might have different styles that might not be reusable. Thinking of making a mixin once there'll be styles that'll be reusable across more then 1 place